<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWidget</class>
 <widget class="QWidget" name="ResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="summaryLabel">
     <property name="text">
      <string>No results loaded. Run a scan or open a results file.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="resultsTree">
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Select</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quality</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Similarity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionsLayout">
     <item>
      <widget class="QLabel" name="selectionLabel">
       <property name="text">
        <string>0 items selected (0.0 MB)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selectAllButton">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deselectAllButton">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectRecommendedButton">
       <property name="text">
        <string>Select Recommended</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>Preview Deletion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete Selected</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #d9534f; color: white; font-weight: bold;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
